README.md

```diff
@@ -109,78 +122,84 @@ by evacuating all citizens with spaceships.
 
-right:
+right click:
  * right click on mainscreen to show info on building under cursor.
- * select tool from menu root
+<!-- * select tool from menu root -->
  * show help on tool
 
-middle:
+middle click:
  * pan map on mainscreen.
 
-left:
+left click:
  * Perform action depending on selected tool. Bulldoze, show Information, construct building.
  * open Tool-Selection-Menus
  * select tool from menus
 
-wheel:
+scroll wheel:
  * up: zoom in
  * down: zoom out
 
 motion:
  * move cursor near the screen edge to scroll main screen
 
-Keyboard
-
- * ESCAPE switch to query tool
 
+#### Keyboard
 
  * KP_PLUS: zoom in
- * KP_MINUS zoom out
- * KP_ENTER zoom 100%
-
-
- * KP5 put middle of the map in the center of the main screen
-
-
- * KP9 scroll main screen north
- * KP1 scroll main screen south
- * KP7 scroll main screen west
- * KP3 scroll main screen east
+ * KP_MINUS: zoom out
+ * KP_ENTER: zoom 100%
 
+ * KP5: center screen on map
 
- * KP2 scroll main screen SE
- * KP4 scroll main screen SW
- * KP6 scroll main screen NE
- * KP8 scroll main screen NW
+ * KP9: scroll main screen north
+ * KP1: scroll main screen south
+ * KP7: scroll main screen west
+ * KP3: scroll main screen east
 
+ * KP2: scroll main screen SE
+ * KP4: scroll main screen SW
+ * KP6: scroll main screen NE
+ * KP8: scroll main screen NW
 
- * arrow keys and WASD: more ways to scroll
+ * arrow keys and WASD: more ways to scroll the main screen
 
+ * SHIFT: scroll main screen faster
 
- * use shift key to scroll faster
 
+ * ESCAPE: switch to query tool
+ * h: hide high buildings. Press h again to show them.
+ * v: cycle through MiniMap-overlay modes
+ * b: toggle between current tool and bulldoze mode
+ * g: toggle between right click showing building or tile info
+ * F1: Help
 
- * h  to hide high buildings. Press h again to show them.
- * v  to cycle through MiniMap-overlay modes
- * b  toggle between current tool and bulldoze mode
- * g  toggle between right click showing building or tile info
- * F1 Help
+ * F12: quick save
+ * F9:  quick load
 
- * F12 quick save
- * F9  quick load
+ * \` (backtick): minimap outline
+ * 1: minimap nourishments
+ * 2: minimap unemployment
+ * 3: minimap power
+ * 4: minimap fire cover
+ * 5: mimimap cricket cover
+ * 6: minimap health cover
+ * 7: minimap traffic
+ * 8: minimap pollution
+ * 9: minimap coal
+ * 0: minimap commodity inventory
 
 ## Contact
 
-Please report bugs and suggest features to:
-    https://github.com/lincity-ng/lincity-ng/issues
+Please report bugs and suggest features to
+    https://github.com/lincity-ng/lincity-ng/issues.
 
 You can contact us at the lincity-ng-devel mailinglist:
-    http://lists.fedorahosted.org/mailman/listinfo/lincity-ng-devel
-or you might be able to catch us in irc at irc.freenode.net #lincity.
+    http://lists.fedorahosted.org/mailman/listinfo/lincity-ng-devel.
+Or you might be able to catch us in irc at irc.freenode.net #lincity.
```

Comment:
Are these still relevant?

Reply:
Well, I'm not on either the mailing list or the IRC, but IDK maybe someone else is.

Though judging from the fact that this project was dormant for so long, I'd guess they are no longer relevant.